### PR TITLE
Add script to generate inline Fizz runtime

### PR DESCRIPTION
**Pull request by acdlite**
This adds a script to generate the inline Fizz runtime. Previously, the runtime source was in an inline comment, and a compiled version of the instructions were hardcoded as strings into the Fizz implementation, where they are injected into the HTML stream.

I've moved the source for the instructions to a regular JavaScript module. A script compiles the instructions with Closure, then generates another module that exports the compiled instructions as strings.

Then the Fizz runtime imports the instructions from the generated module.

To build the instructions, run:

```sh
yarn generate-inline-fizz-runtime
```

The generated Fizz runtime is checked into source. I added a CI check to ensure it stays in sync, by running the script and confirming nothing changed.

### CODE REVIEW

File: packages/react-dom-bindings/src/server/fizz-instruction-set/ReactDOMFizzInlineCompleteSegment.js [hunk 1]
```diff
@@ -0,0 +1,5 @@
+import {completeSegment} from './ReactDOMFizzInstructionSet';
+
+// This is a string so Closure's advanced compilation mode doesn't mangle it.
```
Comment (gaearon) [inline packages/react-dom-bindings/src/server/fizz-instruction-set/ReactDOMFizzInlineCompleteSegment.js:3]: nit: I don’t believe we’re using the “advanced” GCC mode.

Comment (acdlite) [reply packages/react-dom-bindings/src/server/fizz-instruction-set/ReactDOMFizzInlineCompleteSegment.js:3]: We are in the script I just added 😆

**Comment by acdlite** [comment]
There's some subtle thing that's broken in the completeBoundaryWithStyles function. Investigating.

**Review by gnoff — APPROVED** [review]
This is very cool. The one thing that I found helpful when developing the new instruction script was having a non-minified version I could quickly edit and see the output in tests. Is it possible to have a filewatcher run the generation script on save of `ReactDOMFizzInstructionSet.js`?

**Comment by acdlite** [comment]
@sebmarkbage had an idea that once the the external runtime option that @mofeiZ is adding (https://github.com/facebook/react/pull/25437) is complete, we could load the module from source when running Jest tests, which could be nice for local debugging. (Though it wouldn't work if you need to debug an issue specific to the default inline script runtime.)

**Review by sebmarkbage — APPROVED** [review]

**Pull request merged**